spec/models/refinery/jobs/job_application_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'spec_helper'
+require 'helper/generator_helper'
+
+module Refinery
+  module Jobs
+    describe JobApplication, type: :model do
+
+      let(:job_application) {FactoryGirl.create(:job_application)}
+      let(:job) {FactoryGirl.create(:job)}
+
+      describe 'validations' do
+        it 'should be functional' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.